website/server/models/group.js

```diff
@@ -1708,8 +1846,9 @@ schema.methods.checkChatSpam = function groupCheckChatSpam (user) {
 schema.methods.hasActiveGroupPlan = function hasActiveGroupPlan () {
   const now = new Date();
   const { plan } = this.purchased;
-  return plan && plan.customerId
-    && (!plan.dateTerminated || moment(plan.dateTerminated).isAfter(now));
+  return (
+    plan && plan.customerId && (!plan.dateTerminated || moment(plan.dateTerminated).isAfter(now))
+  );
```

Comment:
this looks like an awful lot of irrelevant code style updates--is any of this relevant to achievements? If not, we can probably skip this file. If so, it might work better to just include the achievement-related pieces.